skimage/metrics/simple_metrics.py

```diff
@@ -158,3 +161,99 @@ def peak_signal_noise_ratio(image_true, image_test, *, data_range=None):
 
     err = mean_squared_error(image_true, image_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def _pad_to(arr, shape):
+    """Pad an array with trailing zeros to a given target shape.
+
+    Parameters
+    ----------
+    arr : ndarray
+        The input array.
+    shape : tuple
+        The target shape.
+
+    Returns
+    -------
+    padded : ndarray
+        The padded array.
+
+    Examples
+    --------
+    >>> _pad_to(np.ones((1, 1), dtype=int), (1, 3))
+    array([[1, 0, 0]])
+    """
+    if not all(s >= i for s, i in zip(shape, arr.shape)):
+        raise ValueError(f"Target shape {shape} must be greater than input"
```

Comment:
```suggestion
        raise ValueError(f"Target shape {shape} must be greater than or equal to input"
```
`>`: greater than
`≥`: greater than or equal to
Alternatively, "cannot be less than input shape..."